docs/api_static/plasmapy.analysis.nullpoint.rst

```diff
@@ -6,3 +6,10 @@
 .. currentmodule:: plasmapy.analysis.nullpoint
 
 .. automodapi:: plasmapy.analysis.nullpoint
+
+Example notebooks
+-----------------
+
+.. nbgallery::
+
+   /notebooks/analysis/nullpoint
```

Comment:
I keep forgetting that it's helpful to include a thumbnail gallery of example notebooks in the documentation pages for different modules.